src/lib.rs

```diff
@@ -99,6 +99,26 @@ pub struct GcpImage {
     pub name: String,
 }
 
+/// ReplicatedObject storage images for clouds like IBMCloud/PowerVS
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct ReplicatedObject {
+    /// Mapping from region name to the object.
+    pub regions: HashMap<String, RegionObject>,
+}
+
+/// RegionObject like the ones used for IBMCloud/PowerVS platform containing object, bucket and url .
+#[derive(Debug, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+pub struct RegionObject {
+    /// The actual object in the cloud object storage (.ova.gz for PowerVS, .qcow2 for IBMCloud)
+    pub object: String,
+    /// The bucket where the object resides.
+    pub bucket: String,
+    /// The url of the object.
+    pub url: String,
+}
```

Comment:
Also needs a `release` field per https://github.com/coreos/stream-metadata-go/pull/35#discussion_r740729901.

Reply:
added. also added it to `GcpImage`

Reply:
ah..just noticed you have a PR for GCP, so i'm removing that change.